src/vs/base/node/pfs.ts

```diff
@@ -54,6 +54,16 @@ export function stat(path: string): TPromise<fs.Stats> {
 	return nfcall(fs.stat, path);
 }
 
+export function statLink(path: string): TPromise<{ stat: fs.Stats, isSymbolicLink: boolean }> {
```

Comment:
@isidorn why is this method not simply using extfs? E.g. it should be as simple as:

```typescript
export function statLink(path: string): TPromise<string> {
	return nfcall(extfs. statLink, path);
}
```

Reply:
@bpasero you are correct. I was not aware I can depend on `extfs` in `pfs`, but now I see it is already done